Rework initial scope and deliverables (#3)
OR13/diem-draft-charter: diem-unified-charter.md

# Initial Scope

The working group will focus its initial work on assets that are discovered by use
of digital communication such as use of DNS or at a well-know location on a host identified
by a hostname or IP address. These discovery mechansisms have the
unique challenge of the validator remaining unknown to the operator of the asset.
Specifically the design of other discovery mechanisms using proximity-based protocols such as  QRCodes, NFC, or Bluetooth
is out-of-scope.

Other use cases for future consideration may also cover assets that lack this requirement
or even require separate digital discovery. The WG will ensure maximum reuse of work
for emblem representation and binding. A discovery
mechanism for the initial work will only be specified by this group after the initial emblem
binding protocol is completed (see the Deliverables section below).

> The working group will coordinate with relevant working groups within the IETF and other SDOs to facilitate re-use of existing protocols and capabilities and ensure that existing standards are leveraged appropriately.

Author: ```suggestion
The working group will facilitate re-use of existing protocols and capabilities and ensure that existing standards are leveraged appropriately.
```

1. Use cases and requirements for the initial target scenarios (Informational)
   Initially, the working group will develop and maintain an informational use cases and requirements document, but is not required to request publication of this document. This document must describe at least one digital emblem suitable for indicating protection under international humanitarian law and at least one digital emblem suitable for indicating the presence of a hazard or a certification of the absence of a hazard. This work must assume any specific serialization format or discovery mechanisms.
   As part of this document, the working group will research, analyze and evaluate application layer serialization formats focusing on data structures that match discovery protocols used in the initial scope, such as DNS records or structured fields in DNS, as well as the respective securing mechanisms for digital emblems. The group is not required to describe object level securing mechanisms, in the case that existing protocol level securing mechanisms are sufficient, for example DNSSEC for DNS records, or TLS for well known HTTPS URLs. 

> 2. Architecture document containing a taxonomy, data model, and overall information flows (Informational)

Author: ```suggestion
2. An architecture containing a taxonomy, data model, and overall information flows (Informational)
```

   The working group will develop a high level architecture and data model but start this work only after the use case and requirements document has reached group consensus. The architecture must not assume any specific serialization formats or securing and discovery mechanisms.

3. A protocol specification describing the binding of emblems to assets (Proposed Standard or Experimental)

>    After the architecture and data model are finished, the working group will specify a single concrete serialization for digital emblems and emblems types for one or two of the initial use cases . This document may describe a mandatory to implement securing mechanism. If a securing mechanism is described, this document must describe at least one mandatory to implement cryptographic algorithm which is already supported by the securing mechanism. This protocol is intended for proposed standard, and must not assume any specific discovery mechanisms.
> 
> 4. A protocol specification describing an DNS-based emblem discovery mechanism (Proposed Standard
> or Experimental)
>   A proposed standard describing the use of the DNS as the initial presentation/discovery mechanism for digital emblems. This document may describe a mandatory to implement securing mechanism.

Author: ```suggestion
   After the architecture and data model are finished, the working group will specify a single concrete serialization for digital emblems and emblems types for one or two of the initial use cases. This specification may describe a mandatory to implement securing mechanism. If a securing mechanism is described, at least one mandatory to implement cryptographic algorithm which is already supported by the securing mechanism must be described as well. This protocol specification is intended for proposed standard or experimental, and must not assume any specific discovery mechanisms.

4. A specification describing an DNS-based emblem discovery mechanism (Proposed Standard
or Experimental)
  A proposed standard or experimental specification describing the use of the DNS as the initial presentation/discovery mechanism for digital emblems. This specification may describe a mandatory to implement securing mechanism.
```
